dao/mongo.go

```diff
@@ -282,7 +283,12 @@ func (m *MongoService) GetPaymentsWithRefundPendingStatus() ([]models.PaymentRes
 	collection := m.db.Collection(m.CollectionName)
 	statusFilter := bson.M{refundStatus: "pending"}
 
-	paymentDBResources, err := collection.Find(context.Background(), statusFilter)
+	options := options.Find()
+	options.SetSort(bson.M{"_id": 1})
```

Comment:
Would be better to sort by `refunds.created_at` (if possible) as the creation order of the payment is not the same as the order of refund creations.

Reply:
I left at it is because mongodb has got mechanism to sort by its _id because it has created time encoded together in the _id as well. Moreover it was tested and was doing exactly as wanted that is LIFO.